src/styles/Header.css

```diff
@@ -0,0 +1,8 @@
+.header-image img {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 50px;
+  height: auto;
+  width: 350px; ;
+}
```

Comment:
Did you want to put the image in the center? In that case you need to use display: flex on the parent div ( .header-image{} ). There is also a stray ; on line 7.

You could probably do this, if you want it centered: 

.header-image {
  display: flex;
  justify-content: center;
  align-items: center;
}

.header-image img {
  margin-top: 50px;
  height: auto;
  width: 350px; 
}

/Martin & Etna

Reply:
nope we didn't want it center 🙈